metaci/release/migrations/0014_implementationstep_implementation_step_id.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 3.0.3 on 2021-02-25 18:38
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("release", "0013_implementationstep"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="implementationstep",
+            name="implementation_step_id",
```

Comment:
How is it that you have a migration adding this field but it's not in models.py? Maybe call it "external_id" intead of "implementation_step_id" to avoid repeating the model name and to distinguish it from the primary key in the django db.

Reply:
Ah apologies my code isn't pushed @davisagli